week3/src/components/Card.jsx

```diff
@@ -0,0 +1,58 @@
+import styled from 'styled-components';
+// import theme from "./theme"
+
+
+export default function Card({ data, handleClick }) {
+    const { id, name, img, status, idx } = data
+
+    const handleCard = (idx) => {
+        if (handleClick) handleClick(idx)
+      }
+    
+    return (
+        <CardLayout onClick={() => status || handleCard(idx)}>
+          <CardPiece checked={status}>
+            <CardBack><img src='src/assets/images/KBO.png' width='100%' height='100%'/>
+            </CardBack>
+            <CardFront>
+                <img src = {img} width='100%' height='100%' />
+            </CardFront>
+          </CardPiece>
+        </CardLayout>
+      )
+    }
+    
+const CardLayout = styled.div`
+    display: inline-block;
+    height: 150px;
+    perspective: 600px;
+    width: 9rem;
+    height: 12rem;
+`
+
+const CardPiece = styled.div`
+    width: 100%;
+    height: 100%;
+    position: relative;
+    transition: transform 0.8s;
+    transform-style: preserve-3d;
+    ${({ checked }) => (checked ? `transform: rotateY(180deg)` : null)}
```

Comment:
저는 의미없이 빈 태그와 null, undefined를 return하는 것은 좋지 않은 코드라고 생각을 해요!!
여기도 단락평가를 이용하면 다음과 같이 null을 반환하지 않고도 깔끔하게 코드를 작성할 수 있어요!!
```suggestion
    ${({ checked }) => checked && `transform: rotateY(180deg)`}
```

Reply:
헉 저도 항상 이 방식을 사용했는데 이런 방법도 있었군요! 하나 배워갑니다!!